### PR TITLE
create BAT version of Windows installer

### PR DESCRIPTION
**Pull request by jeffhostetler**
Create BAT file version of sample installer for Windows. This will use a developer command prompt (and a little PowerShell) to create a ZIP file of the component pieces. Inside the ZIP file will be BAT files to install, register, and unregister the collector with the system.

This is intended to replace the original Git-SDK-based GNU Makefile and bash scripts that I used during development, which I've marked "deprecated".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,19 +155,50 @@ stop and restart the service. Use these if you want to try different settings
 in your installed `config.yml` or other YML files (such as `filter.yml`).
 
 
-### On Windows
+### On Windows (Command Prompt)
+
+To build a ZIP file for Windows using a Developer Command Prompt and create
```
Comment (mjcheetham) [inline README.md:160]: "Developer Command Prompt" here means the Visual Studio Developer Command Prompt, and not a 'regular' Windows cmd.exe prompt window, right? May be worth calling this out explicitly.. that it requires the **VS** specific setup.

File: installers/windows_bash/register.sh [hunk 2]
```diff
@@ -57,6 +57,6 @@ $SC start $MyAppName || exit 1
 
 # Tell Git to send Trace2 telemetry to us.
 
-git config --system trace2.configparams $MyNamespace
-git config --system trace2.eventtarget $MyPipe
-git config --system --list --show-origin | grep -i trace2
+git config --global trace2.configparams $MyNamespace
```
Comment (mjcheetham) [inline installers/windows_bash/register.sh:60]: Obviously using `--global` will make configuration per-user, but installation of the service/daemon is per-machine?

File: README.md (hunk 1 above)
Comment (jeffhostetler) [reply README.md:160]: Good point.  TBH I haven't tried it in a plain cmd prompt.  Hey, it works with a plain one.  I clarify it.  Thanks!

**Review by dscho — APPROVED** [review]
Note that Visual Studio's system config includes Git for Windows', IIRC

**Pull request merged**